ext/fetch/lib.rs

```diff
@@ -650,13 +677,17 @@ where
 
 /// Create new instance of async reqwest::Client. This client supports
 /// proxies and doesn't follow redirects.
+#[allow(clippy::too_many_arguments)]
```

Comment:
Maybe time to create some optional params struct? Though I'm guessing that might've existed at some point and been removed for performance reasons?

Reply:
Never has existed; will add one